src/FileConventions/Library.fs

```diff
@@ -60,3 +60,21 @@ let DetectAsteriskInPackageReferenceItems(fileInfo: FileInfo) =
         )
 
     asteriskInPackageReference.IsMatch fileText
+
+let DetectMissingVersionsInNugetPackageReferences(fileInfo: FileInfo) =
+    assert (fileInfo.FullName.EndsWith(".fsx"))
+
+    let fileLines = File.ReadLines fileInfo.FullName
+
+    not(
+        fileLines
+        |> Seq.filter(
+            function
+            | line -> line.StartsWith "#r \"nuget:"
+        )
+        |> Seq.filter(
+            function
+            | line -> not(line.Contains ",")
+        )

```

Comment:
@realmarv I'd prefer if you use `fun` instead of `function`, I'm not a fan of the latter

Reply:
Done